src/backend/distributed/commands/utility_hook.c

```diff
@@ -731,12 +732,29 @@ ExecuteDistributedDDLJob(DDLJob *ddlJob)
 		}
 		PG_CATCH();
 		{
-			ereport(ERROR,
-					(errmsg("CONCURRENTLY-enabled index command failed"),
-					 errdetail("CONCURRENTLY-enabled index commands can fail partially, "
-							   "leaving behind an INVALID index."),
-					 errhint("Use DROP INDEX CONCURRENTLY IF EXISTS to remove the "
-							 "invalid index, then retry the original command.")));
+			MemoryContextSwitchTo(savedContext);
+			ErrorData *edata = CopyErrorData();
+
+			/*in concurrent index creation */
```

Comment:
We prefer a single block of /* */ for multi-line comments. I guess the reason is that it is more readable, especially when there are many lines.

/*
 *
 *
 */